packages/component/src/Activity/ScrollToEndButton.js

```diff
@@ -9,75 +9,63 @@ import useStyleSet from '../hooks/useStyleSet';
 
 const { useDirection, useLocalizer } = hooks;
 
-const ScrollToEndButton = forwardRef(
-  (
-    { 'aria-valuemax': ariaValueMax, 'aria-valuemin': ariaValueMin, 'aria-valuenow': ariaValueNow, className, onClick },
-    ref
-  ) => {
-    const [{ scrollToEndButton: scrollToEndButtonStyleSet }] = useStyleSet();
-    const [direction] = useDirection();
-    const localize = useLocalizer();
-    const scrollToEnd = useScrollToEnd();
+const ScrollToEndButton = forwardRef(({ className, onClick }, ref) => {
+  const [{ scrollToEndButton: scrollToEndButtonStyleSet }] = useStyleSet();
+  const [direction] = useDirection();
+  const localize = useLocalizer();
+  const scrollToEnd = useScrollToEnd();
 
-    const handleClick = useCallback(
-      event => {
-        onClick && onClick(event);
-        scrollToEnd();
-      },
-      [onClick, scrollToEnd]
-    );
+  const handleClick = useCallback(
+    event => {
+      onClick && onClick(event);
+      scrollToEnd();
+    },
+    [onClick, scrollToEnd]
+  );
 
-    const handleKeyPress = useCallback(
-      event => {
-        if (event.key === 'Enter' || event.key === ' ') {
-          event.preventDefault();
+  const handleKeyPress = useCallback(
+    event => {
+      if (event.key === 'Enter' || event.key === ' ') {
+        event.preventDefault();
 
-          onClick && onClick(event);
-          scrollToEnd();
-        }
-      },
-      [onClick, scrollToEnd]
-    );
+        onClick && onClick(event);
+        scrollToEnd();
+      }
+    },
+    [onClick, scrollToEnd]
+  );
 
-    const newMessageText = localize('TRANSCRIPT_NEW_MESSAGES');
+  const newMessageText = localize('TRANSCRIPT_NEW_MESSAGES');
 
-    return (
-      <li
-        aria-label={newMessageText}
-        aria-valuemax={ariaValueMax}
-        aria-valuemin={ariaValueMin}
-        aria-valuenow={ariaValueNow}
-        className={classNames(
-          'webchat__scrollToEndButton',
-          scrollToEndButtonStyleSet + '',
-          className + '',
-          direction === 'rtl' ? 'webchat__overlay--rtl' : ''
-        )}
-        onClick={handleClick}
-        onKeyPress={handleKeyPress}
-        ref={ref}
-        // iOS VoiceOver does not support role="separator" and treat it as role="presentation", which become invisible to VoiceOver.
-        role={safari ? undefined : 'separator'}
-        tabIndex={0}
-      >
-        {newMessageText}
-      </li>
-    );
-  }
-);
+  return (
+    <div
```

Comment:
In light of recent a11y feedback, I'm concerned about changing this to `<div>`. Could you explain the decision a bit?